flow2ml/Flow.py

```diff
@@ -202,3 +204,15 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self):
```

Comment:
You are calling this method having model as param in Readme, but here I don’t see param here?

Reply:
Yes, we need to give the model as input. But there is no model given as an input parameter ?

Reply:
Checking that.